Update nix-tools rev

rvl opened the pull request:
This includes the latest master branch of input-output-hk/nix-tools.

Unfortunately it does not build any more due to:
```
error: attribute 'cabal-doctest' missing, at /nix/store/52nv6czvhhq4ymjij6aymyna5c87blaz-hackage-exprs-source/hackage/distributive-0.6-r0-26507cf231eb10db1eb54bc358477418ce87d5077ff76c66743925fb49494b54.nix:19:49
(use '--show-trace' to show detailed location information)
```

One of the **setup-depends** of `distributive-0.6` is `cabal-doctest`.

michaelpj commented:
This doesn't look quite right - it has generated files without the error functions bound at the top level, which I added later.

michaelpj commented:
Hm, there was something weird with that PR where the commits weren't showing up, they seem to be there now.

michaelpj commented:
No idea what's up with the setup-depends issue.

rvl commented:
Yes that was weird. The PR was definitely not showing the correct commit. I am happy that it's using top-level binds now.

I think the setup-depends build problem is a bug in Haskell.nix.

hamishmack commented:
Check where `cabal new-configure` is being used to generate `plan.json`.  You might need to add `--enable-tests` to get it to include `cabal-doctest` in the plan.

hamishmack commented:
If `--enable-tests` is the culprit here then #225 is probably a good idea too.

rvl commented:
I have rebased and updated the nix-tools revision to latest master. It is still not building and I'm not sure why.

```
rodney@blue:~/iohk/nix-tools % rm -rf dist-newstyle && cabal v2-update && cabal v2-configure --enable-tests
...
rodney@blue:~/iohk/nix-tools % cd ../haskell.nix/nix-tools
rodney@blue:~/iohk/haskell.nix/nix-tools % ../nt/bin/plan-to-nix -o . --plan-json ../../nix-tools/dist-newstyle/cache/plan.json
rodney@blue:~/iohk/haskell.nix/nix-tools % cd ..
rodney@blue:~/iohk/haskell.nix % nix-build -A nix-tools -o nt
error: Neither the Haskell package set or the Nixpkgs package set contain the package: cabal-doctest (build tool dependency).

If this is a system dependency:
You may need to augment the system package mapping in haskell.nix so that it can be found.

If this is a Haskell dependency:
If you are using Stackage, make sure that you are using a snapshot that contains the package. Otherwise you may need to update the Hackage snapshot you are using, usually by updating haskell.nix.

(use '--show-trace' to show detailed location information)
```

I needed to make a manual fix to `nix-tools/pkgs.nix` for it to work.
